application/F3DStarter.cxx

```diff
@@ -858,7 +858,10 @@ class F3DStarter::F3DInternals
       f3d::window& window = this->Engine->getWindow();
       if (this->AppOptions.Resolution.size() == 2)
       {
-        window.setSize(this->AppOptions.Resolution[0], this->AppOptions.Resolution[1]);
+        double dpiScale = f3d::utils::getDPIScale();
+
+        window.setSize(static_cast<int>(this->AppOptions.Resolution[0] * dpiScale),
+          static_cast<int>(this->AppOptions.Resolution[1] * dpiScale));
```

Comment:
maybe a a small doc about this beahvior in https://f3d.app/docs/next/user/DESKTOP_INTEGRATION ?